packages/react-storage/src/components/StorageBrowser/controls/types.ts

```diff
@@ -34,13 +34,15 @@ export interface ControlsContext {
   data: {
     taskCounts?: TaskCounts;
     tableData?: TableData;
+    isDataRefreshDisabled?: boolean;
   };
-  actionsConfig: {
+  actionsConfig?: {
```

Comment:
Updated to optional
cc @Samaritan1011001 @calebpollman @cshfang

Reply:
Still unclear why this is needed at all 😅